pkg/finder/podresourcefinder.go

```diff
@@ -0,0 +1,147 @@
+package finder
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/davecgh/go-spew/spew"
+
+	"k8s.io/api/core/v1"
+
+	podresources "k8s.io/kubernetes/pkg/kubelet/apis/podresources"
+	podresourcesapi "k8s.io/kubernetes/pkg/kubelet/apis/podresources/v1alpha1"
+)
+
+const (
+	defaultPodResourcesTimeout = 10 * time.Second
+	defaultPodResourcesMaxSize = 1024 * 1024 * 16 // 16 Mb
```

Comment:
Please document how we decided these values (is fine if this is an initial estimation, but just document that)

Reply:
Obtained them from [here](https://github.com/kubernetes/kubernetes/blob/82baa26905c94398a0d19e1b1ecf54eb8acb6029/test/e2e_node/util.go#L70). Can add a comment in the code referring to this link